configuration/actions.md

```diff
@@ -263,7 +263,7 @@ Action | Returns
 `getNextBankHoliday` | name of the next bank holiday
 `getNextBankHoliday(<file>)` | name of the next bank holiday defined in `<file>`
 `getNextBankHoliday(<offset>)` | name of the next bank holiday after `<offset>` days from today
-`getNextBankHoliday(<offset>, <file>)` | name of the next bank holiday after `<offset>` days from today defined in `<file>`. :warning: This action is broken in OH 2.5.x. Use `getNextBankHoliday(<datetime>, <file>)` instead by replacing `<datetime>` with `new DateTimeType().zonedDateTime.now().plusDays(<offset>)`
+`getNextBankHoliday(<offset>, <file>)` | name of the next bank holiday after `<offset>` days from today defined in `<file>`. :warning: This action is broken in OH 2.5.x. Use `getNextBankHoliday(<datetime>, <file>)` instead by replacing `<datetime>` with `now.plusDays(<offset>)`
```

Comment:
@jp-Ll, I found this in a6740ab22ebb02d33405f45af3302db47422dfc1, but I'm not sure if it's correct now (or ever was):
- `new DateTimeType()` return a `DateTimeType` with the value of 'now'.
- `.zonedDateTime` extract the `ZonedDateTime` from it, i.e. same as `ZonedDateTime.now()`.
- `.now()` - this is the part I don't get: there is no such method.
- `.plusDays(<offset>)` - self-explanatory

It seems like the idea is to add number of days to 'now' as `ZonedDateTime`. If this is correct, no `DateTimeType` is needed at all. This seems to match:

https://github.com/openhab/openhab-core/blob/e078a5cd982a615a93c60957fa6e25cab43a7734/bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Ephemeris.java#L158-L166